provider/equinix/instance.go

```diff
@@ -52,12 +52,12 @@ func (device *equinixDevice) Status(ctx context.ProviderCallContext) instance.St
 	var jujuStatus status.Status
 
 	switch device.State {
-	case "provisioning":
+	case "provisioning", "queued":
 		jujuStatus = status.Pending
 	case "active":
 		jujuStatus = status.Running
-	case "shutting-down", "terminated", "stopping", "stopped":
-		jujuStatus = status.Empty
+	case "deleted", "deprovisioning", "failed", "inactive":
+		jujuStatus = status.ProvisioningError
```

Comment:
I think these should be typed strings, making it easier to reason about.

Reply:
Can you elaborate what you're looking for here? Maybe an example from another provider. I saw the Azure one but I actually find it much harder to read/reason than what we have here.

Reply:
LXD is a good example here: https://github.com/juju/juju/blob/develop/provider/lxd/instance.go#L38-L56

Reply:
Ok, so unlike LXD, the packngo library doesn't have an enum for these statuses. So I'll need to add one to this file.